.woke.yaml

```diff
@@ -0,0 +1,187 @@
+ignore_files:
+  - abi-compat/*.xml
+  - tests/generator/test_vrfs.py
+  - tests/generator/test_passthrough.py
+  - tests/parser/test_keyfile.py
+  - tests/generator/test_bridges.py
+  - tests/generator/test_bonds.py
+  - tests/generator/test_tunnels.py
+  - tests/generator/test_common.py
```

Comment:
I thought about ignoring some of the tests as well (for now). But OTOH, now is the time to fix everything `woke` finds... It's only ~60 lines which need fixing left, so I think we should put in the extra effort to fix those instances as well.

We might need to adopt the `assert_nm()`, `assert_networkd()` and `generate_keyfile()` methods in their corresponding `base.py` class, but I think it's worth the effort. Like having a simple RegEx string-replace to search for something like `  # wokeignore:rule...` and replace it with an empty string in the corresponding keyfile/networkd assertion function, before comparing it to the actually generated NM/networkd config. This way we should be able to put our "wokeignore" comments into the test's code, while staying compatible with the generated output.